README.md

```diff
@@ -15,37 +15,34 @@ step-by-step tutorial, [visit this link](https://twilio.com/docs/howto/click-to-
 
 [Read the full tutorial here](https://www.twilio.com/docs/tutorials/walkthrough/click-to-call/node/express)!
 
-## Configuration
+### Local Development
 
-### Setting Your Environment Variables
+Getting your local environment setup to work with this app is easy.
 
-Are you using a bash shell? Use echo $SHELL to find out. For a bash shell, using the Gmail example, edit the ~/.bashrc or ~/.bashprofile file and add:
+1. Copy the `.env.example` file to `.env`, and edit it including your credentials
+   for the Twilio API (found at https://www.twilio.com/user/account/settings). You
+   will also need a [Twilio Number](https://www.twilio.com/user/account/phone-numbers/incoming).
 
-```bash
-export TWILIO_ACCOUNT_SID=ACxxxxxxxxxxxxxx
-export TWILIO_AUTH_TOKEN=yyyyyyyyyyyyyyyyy
-export TWILIO_NUMBER=+15556667777
-
-```
-
-Are you using Windows or Linux? You can learn more about how to set variables [here](https://www.java.com/en/download/help/path.xml).
-
-### Development
-
-Getting your local environment setup to work with this app is easy.  
-After you configure your app with the steps above use this guide to
-get it going locally.
+    ```bash
+    cp .env.example .env
+    ```
 
 1. Install the dependencies.
 
     ```bash
     npm install
     ```
 
+1. Run tests.
+
+    ```bash
+    npm test
+    ```
+
 1. Launch local development web server.
 
     ```bash
-    node app.js
+    npm start
```

Comment:
In the ngrok section, can you point out something like "This step is important because the application won't work as expected if you run it through localhost." in a similar fashion as [this readme](https://github.com/TwilioDevEd/clicktocall-flask/blob/master/README.md)?